README.md

```diff
@@ -0,0 +1,123 @@
+# prosemirror-suggestcat-plugin
+
+![made by Emergence Engineering](https://emergence-engineering.com/ee-logo.svg)
+
+[**Made by Emergence-Engineering**](https://emergence-engineering.com/)
+
+Basic UI for [prosemirror-suggestcat-plugin](https://github.com/emergence-engineering/prosemirror-suggestcat-plugin) in React.
+
+## Features
+
+![feature-gif](https://suggestcat.com/basic-suggestr-eact-example.gif)
+
+- A slash menu to select and filter commands, implemented with[prosemirror-slash-menu-react](https://github.com/emergence-engineering/prosemirror-slash-menu-react)
+- A button over selection to open the menu
+- An overlay to show/cancel/reject suggestions
+
+## How to use?
+
+- Import `SlashMenuPlugin` from [`prosemirror-slash-menu`](https://github.com/emergence-engineering/prosemirror-slash-menu-react)
+- Import `ProsemirrorSuggestcatPluginReact` and `promptCommands` from [`prosemirror-suggestcat-plugin-react`](https://github.com/emergence-engineering/prosemirror-suggestcat-plugin-react)
+- Add `SlashMenuPlugin` to your editor with `promptCommands`
+- Create dom reference
+- Add component next to your editor div
+
+```tsx
+import { SlashMenuPlugin } from "prosemirror-slash-menu";
+import {
+  promptCommands,
+  ProsemirrorSuggestcatPluginReact,
+  slashOpeningCondition,
+} from "prosemirror-suggestcat-plugin-react";
+
+const Editor: FC = () => {
+  // Needed for re-renders on every tr.
+  const [editorState, setEditorState] = useState<EditorState>();
+  const [editorView, setEditorView] = useState<EditorView>();
+  const editorRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    const state = EditorState.create({
+      doc: schema.nodeFromJSON(initialDoc),
+      plugins: [
+        SlashMenuPlugin(promptCommands, undefined, slashOpeningCondition, true),
```

Comment:
what is this `undefined` and `slashOpeningCondition`?
in `prosemirror-slash-menu` readme it has only one param, no mention for `slashOpeningCondition`
in `prosemirror-slash-menu` readme:
```ts
  SlashMenuPlugin(defaultElements),
```

Reply:
ahh I see it is here [conditions](https://github.com/emergence-engineering/prosemirror-slash-menu/tree/main#opening-conditions)
but the whole section is referred as `Props` which are actually `arguments`
and I would like to see like a function signature - not here, but in that readme
```ts
SlashMenuPlugin: (menuElement: CommandItem | SubMenu, ignoredKeys: string[], openingConditions: OpeningConditions, openInSelection: boolean) => void
```

Reply:
Added this to the slash menu readme
<img width="425" alt="image" src="https://github.com/emergence-engineering/prosemirror-suggestcat-plugin-react/assets/66436916/6f6c78c7-9a50-4e75-8cd2-834ab3f4afbc">